src/Fields/Fields.jl

```diff
@@ -72,6 +72,13 @@ Base.propertynames(field::Field) = propertynames(getfield(field, :values))
 @inline Base.getproperty(field::Field, name::Integer) =
     Field(getproperty(field_values(field), name), axes(field))
 
+Base.@propagate_inbounds function Base.getindex(
+    field::Field{V},
+    i::Integer,
+) where {V <: DataLayouts.VF}
```

Comment:
We probably want an alias for `ColumnField` to be more explicit (see above).

Reply:
Are you suggesting we write this as `getindex(field::ColumnField, i::Integer)` and then just define a `ColumnField` to specify the types we want this to work for?

Reply:
Yes, that way for any changes we just have to update one def. in the future.